hunting-buddy.lic

```diff
@@ -221,20 +221,30 @@ class HuntingBuddy
           # No visible friends in the room and no visible people
           UserVars.friends.each { |friend| Flags.add("room-check-#{friend}", friend) }
           Flags.add('room-check', 'says, ', 'say, ', 'You hear', 'Someone snipes a')
```

Comment:
Use GitHub's "hide whitespace" feature to reduce the overall diff. Most of the lines have been indented within the new case/when statements.

Reply:
Thank you